hw/ip/otbn/data/otbn.hjson

```diff
@@ -143,7 +140,34 @@
         }
       ]
     } // register : start_addr
-
+    { name: "FATAL_CAUSE",
+      desc: '''
+        The cause of a fatal alert.
+
+        The bits of this register correspond to errors that can cause a fatal
+        alert. Software can read these bits to see what went wrong. Once set,
+        these bits cannot be cleared.
+      '''
+      swaccess: "ro",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "0",
+          name: "imem_ecc_error",
```

Comment:
These just got renamed to remove the ECC part, IIRC, so let's double-check before merging (I think the other PR isn't merged yet?)

Reply:
You're right, it's not merged yet. It's a race! :-D (More seriously, I'm very happy to rebase whichever one comes second, but I don't think that renaming here will make merging the other one easier if this one lands first)

Reply:
Hmm, I've rebased on Greg's error code changes but I don't think the names necessarily need to change. What do you think they should become?

Reply:
`imem_fatal_error`/ `dmem_fatal_error` perhaps?

Reply:
The new error codes are ErrCodeFatalImem, ErrCodeFatalDmem and ErrCodeFatalReg (https://docs.opentitan.org/hw/ip/otbn/doc/index.html#error-conditions), removing the ECC part.

Reply:
Since these are the names of fields in `FATAL_ALERT_CAUSE`, let's go with `imem_error` and `dmem_error` (no need to repeat the "fatal" part).